pkg/ebpf/events_pipeline.go

```diff
@@ -427,6 +427,7 @@ func (t *Tracee) sinkEvents(ctx context.Context, in <-chan *trace.Event) <-chan
 		for event := range in {
 			// Only emit events requested by the user
 			id := events.ID(event.EventID)
+			event.MatchedScopes &= t.events[id].emit
 			if (t.events[id].emit & event.MatchedScopes) == 0 {
```

Comment:
You can now simply check:
`if (event.MatchedScopes) == 0`

Reply:
Good catch. Done.